plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3/TrinoS3FileSystem.java

```diff
@@ -205,6 +208,9 @@
     public static final String S3_PROXY_PASSWORD = "trino.s3.proxy.password";
     public static final String S3_PREEMPTIVE_BASIC_PROXY_AUTH = "trino.s3.proxy.preemptive-basic-auth";
 
+    public static final String S3_STS_ENDPOINT = "trino.s3.sts.endpoint";
+    public static final String S3_STS_REGION = "trino.s3.sts.region";
```

Comment:
Currently you are never putting entries for those keys in the `Configuration`.
I think for any configuration key defined here we have matching 1st class Trino config defined in `HiveS3Config`. Should be the same for config properties you add.

Reply:
true, I added entries in `HiveS3Config` and made sure they are used in `TrinoS3ConfigurationInitializer`